frontend_server/lib/server.dart

```diff
@@ -338,9 +338,8 @@ Future<int> starter(
 
   if (options['train']) {
     final String sdkRoot = options['sdk-root'];
-    final String outputTrainingDill = p.join(
-        Directory.systemTemp.createTempSync('train_frontend_server').path,
-        'app.dill');
+    Directory temp = Directory.systemTemp.createTempSync('train_frontend_server');
+    final String outputTrainingDill = path.join(temp.path, 'app.dill');
     options = _argParser.parse(<String>['--incremental', '--sdk-root=$sdkRoot',
```

Comment:
I had a sudden revelation about this code: is `$sdkRoot` still relevant? I thought we got read of `flutter_patched_sdk`?

Reply:
flutter_patched_sdk contains platform.dill files at this point. It used to have compete patched dart sdk sources - that's what we got rid of recently.